app/views/checkout/index.html.haml

```diff
@@ -118,6 +118,16 @@
               %td{ colspan: "4"}
                 %strong
                   Total: {{ total() | currency }}
+        .alert.alert-danger{ ng: { if: 'couponErrors' }}
+          %ul{ ng: { repeat: "val in couponErrors" }}
+            %li
+              {{ val }}
+        .col-sm-3
+          %label.control-label Coupon
+        .col-sm-3
+          %input.form-control{ type: "text", class: "form-control", ng: { model: "couponCode"} }
```

Comment:
@gsnovak Sure, we can do that after you remove the redundant `class: "form-control"`

Reply:
whoops